### PR TITLE
Updates to the README and yaml files

### PR DESCRIPTION
**Pull request by rendzina**
Updates to the README and yaml files made to improve readability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
 # DAFNI Model Examples
 
-This repo contains several simple examples of DAFNI ready models.
+This repo contains several simple examples of DAFNI ready models, useful for
+training of self-paced learning.
```
Comment (akademy) [inline README.md]:
```suggestion
training or self-paced learning.
```

File: uk-climate-analysis/README.md [hunk 2]
```diff
@@ -25,13 +25,61 @@ datasets needed to run the model for all feature types are listed below:
 
 • https://data.ceda.ac.uk/badc/ukcp09/data/gridded-land-obs/gridded-land-obs-monthly/grid/ascii/sunshine
 
+Note, these data have already been loaded into DAFNI, so with the appropriate permissions (just ask us) you can access them directly there to save space. Alternatively download the files from CEDA and file them in your folder structure (noting each of these is a zip file, thus in /data/inputs/rainfall you have 'rainfall.zip' etc.) The data will then be incorporate into the dockr image that you upload.
```
Comment (akademy) [inline uk-climate-analysis/README.md]:
```suggestion
Note, these data have already been loaded into DAFNI, so with the appropriate permissions (just ask us) you can access them directly there to save space. Alternatively download the files from CEDA and file them in your folder structure (noting each of these is a zip file, thus in /data/inputs/rainfall you have 'rainfall.zip' etc.) The data will then be incorporate into the docker image that you upload.
```

File: uk-climate-analysis/README.md [hunk 3]
```diff
@@ -25,13 +25,61 @@ datasets needed to run the model for all feature types are listed below:
...
+`docker save -o uk-climate-analysis.tar uk-climate-analysis`
+
+### Compress resultant tar image
+`gzip uk-climate-analysis.tar`
+This compresse gz file, and the yaml file can now be uploaded to DAFNI.
+
 ## Running the Model
+Running the model in DAFNI follows a different path from the text below - the DAFNI `Dockerfile` contains the following two lines:
+
+`RUN pip install -r requirements.txt`<br />
```
Comment (akademy) [inline uk-climate-analysis/README.md]:
```suggestion
`RUN pip install -r requirements.txt`
```

File: uk-climate-analysis/README.md [hunk 4]
```diff
@@ -25,13 +25,61 @@ datasets needed to run the model for all feature types are listed below:
...
+Then, use `python climate_analysis.py` to run the model.
 
+## Visualising the Model Output
 The `visualisation.py` file assumes you have imported the data into a Jupyter
 notebook but you don't have to do this. You can adapt the `get_output_folder()`
-folder to point to your output folder instead.
\ No newline at end of file
+folder to point to your output folder instead.
+
+In DAFNI, you will create a new 'Jupyter notebook' in the visualisation tool and then copy and paste in the ode from the `visualisation.py` file.
```
Comment (akademy) [inline uk-climate-analysis/README.md]:
```suggestion
In DAFNI, you will create a new 'Jupyter notebook' in the visualisation tool and then copy and paste in the code from the `visualisation.py` file.
```

File: uk-climate-analysis/README.md [hunk 5]
```diff
@@ -25,13 +25,61 @@ datasets needed to run the model for all feature types are listed below:
...
+
+Note to get the `visualisation.py` script to work, you may first need to make some edits (and comment out some lines), such as:
+
+### Import section
+import statsmodels.api as sm
+\#from statsmodels.tsa.base.tsa_model import ValueWarning
```
Comment (akademy) [inline uk-climate-analysis/README.md]:
```suggestion
#from statsmodels.tsa.base.tsa_model import ValueWarning
```
There's a few "/#" here and below that need to be removed.

**Review by akademy — CHANGES_REQUESTED** [review]
Looks good, just a few typos.

File: uk-climate-analysis/README.md (hunk 4 above)
Comment (rendzina) [reply uk-climate-analysis/README.md]: Edits made to source and saved back - plus other typos spotted!

**Comment by rendzina** [comment]
Hope edits requested are all OK.

File: uk-climate-analysis/README.md (hunk 3 above)
Comment (rendzina) [reply uk-climate-analysis/README.md]: Changes all fixed in repo

**Review by roseDickinson — APPROVED** [review]

**Comment by akademy** [comment]
Thanks @rendzina for the work you done here. Much appreciated. 

(And thanks to @roseDickinson for merging)

**Comment by rendzina** [comment]
Thanks @akademy - I will add in more in due course, having the example models is great. By the way I think the YAML file directives will need to be updated with the new API now (desc -> description and requirements etc.) Perhaps best to leave that edit to you. Cheers

**Comment by roseDickinson** [comment]
I updated the YAML definition files to work with the latest version of the API yesterday after merging this MR, if there are any outstanding issues with the files please let us know!

**Pull request merged**